src/index.html

```diff
@@ -4,14 +4,203 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset class="form-section">
+
+        <!-- Personal info -->
+
+        <legend>Personal information</legend>
```

Comment:
- Don't add empty lines between elements of different levels of nesting
- don't leave unnecessary comments in your code - this can confuse those who read. Good code is self-documenting, no need for comments unless you really need it
- if the line with element and its attributes is too long - move each of the attributes to a new line - this makes your code easier to read

```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="get"
    >
      <fieldset class="form-section">
        <legend>Personal information</legend>
```